### PR TITLE
Change MySQL definition columns to LONGBLOB in prep for supporting larger definitions

### PR DESCRIPTION
**Pull request by josephschorr**
Part #1 of #1468

While each definition will not likely be super large, there is a definite need for larger individual definitions

### CODE REVIEW

File: internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import "fmt"
+
+func convertNamespaceDefinitionToLongBlob(t *tables) string {
+	return fmt.Sprintf(`ALTER TABLE %s
+		MODIFY serialized_config LONGBLOB NOT NULL;`,
```
Comment (ecordell) [inline internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go:7]: We sure we don't want MEDIUMBLOB here? 

BLOB = 64KB
MEDIUMBLOB=16MB
LONGBLOB=4GB

64k to 4g seems like a big jump

Comment (josephschorr) [reply internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go:7]: It is merely a max size, still constrained by the overall schema size limits. Further, the other types have overhead: https://www.tutorialspoint.com/What-is-the-maximum-length-of-data-we-can-put-in-a-BLOB-column-in-MySQL

Comment (ecordell) [reply internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go:7]: that doc is definitely wrong, longblobs [use a 4-byte prefix](https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#:~:text=L%20%2B%204%20bytes%2C%20where%20L%20%3C%20232) (the only distinction between the types is the number of bytes used for the length prefix)

but the 1 byte difference is probably not a big deal, lgtm

**Review by ecordell — APPROVED** [review]

File: internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go (hunk 1 above)
Comment (josephschorr) [reply internal/datastore/mysql/migrations/zz_migration.0006_convert_definitions_to_longblob.go:7]: Darn internet... maybe we use MEDIUMBLOB then?

**Pull request merged**